homeassistant/components/zha/__init__.py

```diff
@@ -293,6 +325,35 @@ def register_entity(self, ieee, entity_obj):
             self._config,
         )
 
+    async def _async_setup_remote(self, discovery_info):
```

Comment:
should we move this into a separate file, just to keep `__init__.py` less cluttered?